exercises/isbn-verifier/isbn_verifier_test.rb

```diff
@@ -0,0 +1,105 @@
+require 'minitest/autorun'
+require_relative 'isbn_verifier'
+
+# Common test data version: 2.0.0 3251fa6
+class IsbnVerifierTest < Minitest::Test
+  def test_valid_isbn_number
+    # skip
+    string = "3-598-21508-8"
+    assert IsbnVerifier.valid?(string), "Expected true, '3-598-21508-8' is a valid isbn"
```

Comment:
The message given should be guaranteed to be correct.  The `string` should be used in the message to avoid any future problems of 'value changed, but feedback did not'